policy/observers.md

```diff
@@ -0,0 +1,22 @@
+# Observers
+
+**Administrator:** A Working Group in Electron Governance or a company represented in a Working Group wishing to introduce an Observer into their Working Group.
+
+**Temporary worker:** A person employed or otherwise contracted by an Administrator for a temporary and fixed duration. Their role must entail that they primarily work on the Electron framework and/or other responsibilities of the Working Group they wish to join.
+
+
+## Working Group Membership
```

Comment:
I think we should specify where the contractual date can be located. @codebytere had an example: "this role must have a predetermined start and end date, noted in their section of the WG membership list"